opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -1619,7 +1619,7 @@ const updateAttributeRaw = async (context, user, instance, inputs, opts = {}) =>
           }
         }
         // Regenerated the internal ids with the instance target aliases
-        const aliasesId = generateAliasesId(R.uniq([askedModificationName, ...aliases]), instance);
+        const aliasesId = generateAliasesId(aliases, instance);
```

Comment:
why remove askedModificationName ?

Reply:
As alias are now normal id, it will be a duplicate between standard_id and the alias_id generated